timber/build.gradle

```diff
@@ -1,5 +1,7 @@
 apply plugin: 'com.android.library'
 apply plugin: 'org.jetbrains.kotlin.android'
+apply plugin: 'org.jetbrains.dokka'
+apply plugin: 'com.vanniktech.maven.publish'
```

Comment:
You don't need a 3rd party plugin to publish for quite some time now.
https://developer.android.com/studio/build/maven-publish-plugin
https://docs.gradle.org/current/userguide/publishing_maven.html

Reply:
This seems to simplify things a lot, plus I noticed it was being used by some Square libraries.